component-lib/navigation.html

```diff
@@ -0,0 +1,252 @@
+                                    sandbox="allow-scripts allow-same-origin">
+                                </iframe>
+							</code>
```

Comment:
You can give left right margin to your iframe, because on mobile screens it is sticked to both the sides